dev/com.ibm.ws.repository.resolver/src/com/ibm/ws/repository/resolver/RepositoryResolver.java

```diff
@@ -931,7 +1050,16 @@ private void reportErrors() throws RepositoryResolutionException {
             missingRequirementNames.add(req.getRequirementName());
         }
 
-        throw new RepositoryResolutionException(null, missingTopLevelRequirements, missingRequirementNames, missingProductInformation, missingRequirements, featureConflicts);
+        throw new RepositoryResolutionException(null, missingTopLevelRequirements, missingRequirementNames, missingProductInformation, missingRequirements, featureConflicts,
+                                                resolvedPlatforms, missingPlatforms, missingBasePlatforms);
+    }
+
+    /**
+     * @return
+     */
+    protected boolean hasVersionlessIssue() {
+
+        return hasVersionlessIssue;
     }
```

Comment:
Unsure why this protected method is needed.

Reply:
removed